com.hybris.hyeclipse.commons/src/com/hybris/hyeclipse/commons/utils/Constants.java

```diff
@@ -0,0 +1,11 @@
+package com.hybris.hyeclipse.commons.utils;
+
+/**
+ * Useful common Constants
+ */
+public interface Constants {
```

Comment:
Can we name this something more meaningful  ? Like CharatersConstatnts or something.